doc/getting-started.rst

```diff
@@ -24,4 +24,19 @@ Next steps
 - See the example EFD notebooks in the `lsst-sqre/notebook-demo repository`_.
 - Refer to the :ref:`py-api`.
 
+Special Notes
+=============
+
+Retrieving Indexed Component Information
+----------------------------------------
+
+When SAL 7 is released, the field name for indexed components will change to *salIndex* from *{CSCName}ID*.
+The queries in the EFD client have been adjusted to use the new name when passing the *index* parameter.
+A boolean flag, *use_old_csc_indexing* has been added to all queries that allow one to retrieve the old field name.
+Set this flag to `True` in order to get the old indexing scheme.
+The TTS conversion to SAL 7 will take place on June 21, 2022.
+However, the TTS EFD operates on a 30 day rotation, so the older indexing will phase out approximately 30 days after the upgrade happens.
+The summit, and by fiat the LDF replica, will convert to SAL 7 on July 6, 2022.
+Since neither database operates with a retention policy, two separate queries must be constructed in order to get data selected on index spanning the above date.
```

Comment:
Is the mention of a retention policy here just clarifying that there won't be a rolloff when the old index is no longer available?

data selected on index --> data selected on an index ?

Reply:
Yes, the retention policy mention is clarifying. The summit SHOULD be on a retention policy, but it's not yet. The replica will never have one. I'll fix the typo.

Reply:
Typo fixed.